Tests/scripts/run_mypy.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+# Run mypy with a set of default parameters. 
+# Will not return an error code even if mypy returns so. 
+# This allows treating mypy as non-fatal.
+#
+# Arguments: python_version python_file
+#
+# Env: MYPY_NO_FAIL: if set will always return a 0 return code. Can be used in nightly to not fail the build.
+
+if [[ $# -lt 2 ]]; then
+    echo "Usage: $BASH_SOURCE <python_version> <python_files>"
+    echo "For example: $BASH_SOURCE 2.7 Active_Directory_Query.py"
+    exit 1
+fi
+
+PY_VERSION=$1
+PY_FILE=$2
+PY_BACKUP=""
+
+if [[ "$PY_VERSION" = "2.7" ]]; then
+    # typing import if not present
+    grep -E '^(from typing import|import typing)' ServiceNow.py > /dev/null 2>&1
```

Comment:
the `ServiceNow.py` is leftovers?
```suggestion
    grep -E '^(from typing import|import typing)' "$PY_FILE" > /dev/null 2>&1
```